### ISSUE TITLE
[Nov patch 1] Itemconfig Names/Descriptions broken when using non-english languages

### ISSUE THREAD

**Issue by jsgnextortex** [Meowlala/RepentanceAPIIssueTracker#344]
Item descriptions and names are broken when using a non-english name (Ex: spanish). they show up as #NAME_OF_THE_ITEM_NAME and #NAME_OF_THE_ITEM_DESCRIPTION

**Comment by namishere** [comment]
Looks like items now use a localized string that can change based on game language, this bug is likely just a result of returning the raw string in items.xml instead of the current localized string

**Comment by jsgnextortex** [comment]
Yes, that is what is happening. They are not placeholder, just unparsed translate strings.

**Comment by Zamiell** [comment]
In the meantime, you can use:
<https://github.com/IsaacScript/isaacscript-common/blob/main/src/collectibleNameMap.ts>
<https://github.com/IsaacScript/isaacscript-common/blob/main/src/trinketNameMap.ts>
<https://github.com/IsaacScript/isaacscript-common/blob/main/src/cardNameMap.ts>
<https://github.com/IsaacScript/isaacscript-common/blob/main/src/pillEffectNameMap.ts>

Or IsaacScript users can just use the provided helper functions `getCollectibleName`, `getTrinketName`, `getCardName`, and `getPillEffectName`.
